### DISCUSSION TITLE
Difference in spatial mapping methods between tutorials

### DISCUSSION THREAD

**Discussion by meaksu** [BayraktarLab/cell2location#268] [Q&A] [answered]
Hi,
I was wondering if there are multiple methods for performing the spatial mapping part of the pipeline. It looks like in this tutorial:
https://cell2location.readthedocs.io/en/latest/notebooks/cell2location_tutorial.html
you first create a mod object using `cell2location.models.Cell2location()` and then run mod.train on this object. 

However in this tutorial:
https://cell2location.readthedocs.io/en/latest/notebooks/cell2location_short_demo.html
you use the `cell2location.run_cell2location()` function and this requires much more hyperparameters to be defined.

When would you use one of these methods over the other? Thanks!

**Comment by vitkl** [comment, marked as answer]
`cell2location.run_cell2location()` is deprecated and will not work (past v0.1.0). Please don't use https://cell2location.readthedocs.io/en/latest/notebooks/cell2location_short_demo.html. This is a tutorial for the 2020 version of cell2location (<0.1.0). This notebook shows how to combine multiple Visium datasets - but everything else is irrelevant. I will figure out why the documentation website still shows it.
